index.js

```diff
@@ -6,12 +6,39 @@ module.exports = {
   extends: ['airbnb'],
   plugins: ['chai-friendly'],
   rules: {
-    'no-unused-expressions': 0,
-    'chai-friendly/no-unused-expressions': 2,
+    'no-unused-expressions': 'off',
+    'chai-friendly/no-unused-expressions': 'error',
     'linebreak-style': ['error', 'unix'],
     'no-param-reassign': ['error', { props: false }],
     'func-style': ['error', 'declaration', { allowArrowFunctions: true }],
     'no-use-before-define': ['error', { functions: false }],
-    'consistent-return': 0,
+    'no-unused-vars': ['error', {'vars': 'all', 'args': 'none'}],
+    'consistent-return': 'off',
+    'arrow-body-style': ['warn', 'as-needed', {
+      requireReturnForObjectLiteral: false,
+    }],
+    'prefer-rest-params': 'warn',
+    'prefer-destructuring': ['warn', {
+      VariableDeclarator: {
+        array: false,
+        object: true,
+      },
+      AssignmentExpression: {
+        array: true,
+        object: true,
+      },
+    }, {
+      enforceForRenamedProperties: false,
+    }],
+
+    'no-underscore-dangle': 'off',
```

Comment:
allowing for underscores usually leads to confusing naming

Reply:
As of?

Reply:
`user`, `user_`, `user__`
which user do i want

Reply:
I have a very specific use case for this where I'd like to be able to use underscore dangles. In Vue, when you have a computed property that needs to store some data that shouldn't be used anywhere else or in the template. I know I've had this issue somewhere before.

Reply:
I also use it as a return value of a transformation. because savedDojo is a terrible naming, and res isn't strictly better.
I can stick to other naming scheme, like `l_dojo` but that's imho even more cryptic. Welcome to js ?

Reply:
Yes, this is another example where I find I want to use underscore dangles. As long as we don't have long spaghetti functions, then it should be easy to keep track of.